src/sanitise-old-messages/src/sanitiseOldMessages.ts

```diff
@@ -0,0 +1,105 @@
+import { add } from "date-fns"
+import type { Client } from "pg"
+import { logger } from "shared"
+import type { ApiClient, AuditLog, AuditLogDynamoGateway, PromiseResult } from "shared-types"
+import { isError } from "shared-types"
+import type { SanitiseOldMessagesConfig } from "./config"
+
+export default class SanitiseOldMessages {
+  constructor(
+    private api: ApiClient,
+    private dynamo: AuditLogDynamoGateway,
+    private db: Client,
+    private config: SanitiseOldMessagesConfig
+  ) {
+    this.api = api
+    this.dynamo = dynamo
+    this.db = db
+    this.config = config
```

Comment:
[Very minor] Lines 15 to 18 are not needed. `private` access modifier in constructor's parameters define a variable in the class. So there is no need to assign them.

Reply:
Done this 👍